prime-router/cleanslate.sh

```diff
@@ -92,33 +99,39 @@ function error() {
 }
 
 # Takes ownership of some directories that are potentially shared with container instances
+# Parameter: TAKE_OWNERSHIP value
 function take_directory_ownership() {
-  info "Taking ownership of directories (may require elevation)..."
-  TARGETS=(
-    "./build/"
-    "./docs/"
-    "./.gradle/"
-    "./.vault/"
-  )
-
-  # If you have no sudo, then this is a no-op and we warn
-  SUDO_CMD=$(which sudo | head -n 1)
-  if [[ -z "${SUDO_CMD}" ]]; then
-    warning "You do not apear to have 'sudo'; it is highly probable that incorrect file permissions will lead to sftp upload failures when you run tests"
-  fi
+  
+  if [[ ${1} != 0 ]]; then
```

Comment:
cf second comment from original review: consider making this `if [[ ${1:-1} != 0 ]]; then` to make this function be callable without arguments as well

Reply:
Shouldn't it be defaulted to 0 and not 1?

Reply:
no, given the name of the function, we _want_ it to do the action, _unless_ told otherwise (i.e. arg 1 == non-zero)